### PR TITLE
port tracepoint operational test

### PR DESCRIPTION
**Pull request by ftjahjad**
Port Systemtap Tracepoint Operational Test

### CODE REVIEW

**Comment by ftjahjad** [comment]
Hi @veruu , and @rasibley 

Please review the PR.

Thanks,

File: tracepoints/operational/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            rm -f $PROBES_NAME_FILE
+        fi
+    done
+}
+
+function runTest ()
+{
+    local TESTLIST=`/usr/bin/stap -L 'kernel.trace("*")' | grep -o "\".*\""`
+    if [ -z "$TESTLIST" ] ; then
+        resultFail TESTLIST_EMPTY 99
```
Comment (veruu) [inline tracepoints/operational/runtest.sh]: I'm not sure if this is something we should report as a failure or a prerequisite for the testing (in which case we should skip or abort), as I'm not familiar with this at all. @ftjahjad can you check which one it is?

File: tracepoints/operational/runtest.sh [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+# Checking if running kernel-rt
+if [ $(is_kernel_rt) -eq 0 ]; then
+    ${yum} -y install kernel-rt-devel-${kernel}
+else
+    ${yum} -y install kernel-devel-${kernel}
+fi
+res=$?
+if [ $res -ne 0 ] ; then
+    echo "Failed to install kernel devel-${kernel}" | tee -a $OUTPUTFILE
```
Comment (veruu) [inline tracepoints/operational/runtest.sh]: What will the exit code be here? Should an explicit warn be reported?

File: tracepoints/operational/runtest.sh (hunk 1 above)
Comment (jbastian) [reply tracepoints/operational/runtest.sh]: If SystemTap cannot find any kernel tracepoints, I would consider that a failure.  Unless you intend to build a kernel with tracepoints disabled, this should be a failure (not an abort).

Comment (ftjahjad) [reply tracepoints/operational/runtest.sh]: @jbastian , thanks. it is clear now.

File: tracepoints/operational/runtest.sh (hunk 2 above)
Comment (ftjahjad) [reply tracepoints/operational/runtest.sh]: adding "report  warn" before aborting.

**Comment by ftjahjad** [comment]
@veruu 
Thanks for pointing put the missing "warn/aborted"  report. :+1: 

Please  review  changes to report   "warn/aborted"  before aborting the recipe.

File: tracepoints/operational/runtest.sh (hunk 1 above)
Comment (veruu) [reply tracepoints/operational/runtest.sh]: We either build normal tarballs (upstream) or use rpmbuild with the following options:

```--with=kabidw_base --without=bpftool --without=debug --without=doc --without=ipaclones --without=perf --without=selftests --without=tools
```
The configs used are the standard configs used with the release.
So I think the building part should include tracepoints and everything should be ok. Thanks @jbastian !

**Pull request merged**